src/Catalyst.Core.Modules.Keystore.Tests/IntegrationTests/KeyApiTest.cs

```diff
@@ -0,0 +1,354 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Security;
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Cryptography;
+using Catalyst.Abstractions.Keystore;
+using Catalyst.Abstractions.Options;
+using Catalyst.Core.Lib.Config;
+using Catalyst.TestUtils;
+using Lib.P2P;
+using Makaretu.Dns;
+using NSubstitute;
+using Org.BouncyCastle.Crypto.Parameters;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Catalyst.Core.Modules.Keystore.Tests.IntegrationTests
+{
+    public sealed class KeyApiTest : FileSystemBasedTest
+    {
+        private readonly IKeyStoreService _keyStoreService;
+
+        public KeyApiTest(ITestOutputHelper output) : base(output)
+        {
+            var dfsOptions = new DfsOptions(new BlockOptions(), new DiscoveryOptions(), new RepositoryOptions(FileSystem, Constants.DfsDataSubDir), Substitute.For<KeyChainOptions>(), Substitute.For<SwarmOptions>(), Substitute.For<IDnsClient>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 245 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)